x-pack/plugin/deprecation/src/main/java/org/elasticsearch/xpack/deprecation/ClusterDeprecationChecks.java

```diff
@@ -58,7 +60,57 @@ static DeprecationIssue checkUserAgentPipelines(ClusterState state) {
                 "Ingest pipelines " + pipelinesWithDeprecatedEcsConfig + " uses the [ecs] option which needs to be removed to work in 8.0");
         }
         return null;
+    }
+
+    @SuppressWarnings("unchecked")
+    static DeprecationIssue checkFormatOnPipeline(ClusterState state) {
```

Comment:
any ideas how to check that pipeline was created in 6.x?

Reply:
I don't think it's possible to check when the pipeline was created, unfortunately. However, it's much easier to update pipelines to use the `8` prefix (compared to mappings) so I don't think it matters as much here.

Reply:
@gwbrown agree, it is easier to fix them (just a PUT with a fixed pipeline). Should we then alert for all pipelines without the 8prefix, suspecting that it was not updated? 
Or should we not alert at all and let user find this on their own?

Reply:
Users in 7 are not required to prefix their format with 8. 
If we just alert without checking the version, and someone changes their pattern from `Y` to `y` it will still alert from him. Both are deprecated in 6.x. (Y->y and y->u)
I wonder if the best way here is not alert about pipelines at all in 7x

Reply:
we discussed this offline and decided to remove pipelines' checks in 7.x. There would be no easy way to fix these deprecation warnings and this is likely to generate a lot of false positives.